tests/integration-tests/tests/ad_integration/test_ad_integration.py

```diff
@@ -837,3 +837,108 @@ def _check_ssh_auth(user, expect_success=True):
             raise e
         else:
             logging.info(f"SSH access denied for user {user.alias}, as expected")
+
+
+@pytest.mark.parametrize(
+    "directory_type,directory_protocol,directory_certificate_verification",
+    [
+        ("SimpleAD", "ldap", False),
+        ("MicrosoftAD", "ldaps", True),
```

Comment:
I suggest to keep only SimpleAD because the integration between login nodes and AD is equivalent to the one in the head node, which is already covered by both SimpleAd and MsAD, so for login nodes should be enoiugh to test just one. I suggest to use SimpleAd because it takes less time to be deployed.

Reply:
I agree, but I'd keep it.
The long term goal is to move all tests related to regular-users behavior and UX on the LoginNodes.
So this is a step in that direction (and long term we are going to remove the test on the HeadNode and enrich this one).